web/src/installers/versions.js

```diff
@@ -355,6 +355,7 @@ module.exports.InstallerVersions = {
     "1.12.0",
     "1.6.0",
     // cron-openebs-update-3
+    "3.3.1",
```

Comment:
It's worth mentioning and clarifying that this is the version of the Helm chart NOT a new version of the openebs application.